rocky/reports/templates/partials/report_header.html

```diff
@@ -13,7 +13,12 @@ <h1>{{ report_name }}</h1>
                         <li>
                             <a href="{{ report_download_pdf_url }}"
                                target="_blank"
-                               rel="noopener noreferrer"><span class="icon ti-download" aria-hidden="true"></span>{% translate "Download report" %}</a>
+                               rel="noopener noreferrer"><span class="icon ti-download" aria-hidden="true"></span>{% translate "Download report" %} (PDF)</a>
```

Comment:
rel="noopener noreferrer" are only needed when dealing with links to external hosts.